contentcuration/contentcuration/frontend/channelEdit/components/AssessmentItemToolbar.vue

```diff
@@ -15,9 +15,15 @@
             v-on="on"
             @click="clickItem(action)"
           >
-            <Icon v-if="config[action] && config[action].icon" :color="iconColor(action)">
-              {{ config[action].icon }}
-            </Icon>
+            <Icon
+              v-if="config[action] && config[action].icon"
+              :icon="
+                config[action].icon
+              "
+              style="font-size: 20px;"
```

Comment:
@MisRob I wonder if we should consider some size options for `KIcon`, to promote visual consistency (@BabyElias nothing in regards to your work here)

Reply:
@bjester Yes, that'd make sense to me, thank you. I can mention it in the design system meeting and see if designers can advice some most common pattern.

Reply:
I will mark it resolved now. I agree it's something to deal with outside the scope of this PR.